app/helpers/taxjar_helper.rb

```diff
@@ -0,0 +1,70 @@
+module TaxjarHelper
+  class Pretty < Logger::Formatter
+    # Provide a call() method that returns the formatted message.
+    def call(severity, time, program_name, message)
+      "#{time.utc.iso8601} #{Process.pid} TID-#{Thread.current.object_id.to_s(36)}#{context} #{severity}: #{message}\n"
+    end
+
+    def context
+      Thread.current[:spree_taxjar_context] ? " #{c}" : ''
+    end
+  end
+
+  class TaxjarLog
+    def initialize(path_name, file_name, log_info = nil, schedule = nil)
+      schedule = "weekly" unless schedule != nil
```

Comment:
@pikender - We can also write it as `schedule ||= "weekly` or we can directly add it to the default value.